cmd/client.go

```diff
@@ -92,14 +92,24 @@ func (cli *Client) GetJobSpecs(c *clipkg.Context) error {
 func (cli *Client) CreateJobSpec(c *clipkg.Context) error {
 	cfg := cli.Config
 	if !c.Args().Present() {
-		return cli.errorOut(errors.New("Must pass in job JSON"))
+		return cli.errorOut(errors.New("Must pass in JSON/filepath"))
+	}
+	var buf *bytes.Buffer
+	if c.Args().First()[0] != '{' {
+		file, err := ioutil.ReadFile(c.Args().First())
```

Comment:
Let's have support for `~` paths. This is done with `homedir` as show here:

```
	dir, err := homedir.Expand(config.RootDir)
```

https://github.com/smartcontractkit/chainlink/blob/master/store/config.go#L41